composer.json

```diff
@@ -28,7 +28,8 @@
         "php" : ">=7.2",
         "ext-curl": "*",
         "ext-intl": "*",
-        "ext-mbstring": "*"
+        "ext-mbstring": "*",
+        "symfony/process": "^4.3"
```

Comment:
Is it possible replace this by `proc_open` or `popen`?

https://www.php.net/manual/en/function.proc-open.php
https://www.php.net/manual/en/function.popen.php

Reply:
Could be not so sure, but why to not to use the same clean solution that everyone else is using.

Reply:
Because it's +1 dependency requirement due to the feature (not base functionality)
And as I see you just call whois with args and getting result. So native functions does the same from a box..

Reply:
I'll try to see what can I do in the weekend.

Reply:
Ok, no problem.
As I remember it's simple enough. Just like socket/file resource handling

Reply:
Hi
I implemented CliWhoisLoader in v4.
Looks like it works ;)
```
$loader = new \Iodev\Whois\Loaders\CliWhoisLoader();

$whois = Factory::get()->createWhois($loader);

var_dump([
    $whois->lookupDomain('google.com'),
]);
```

Reply:
looks great, and I am really sorry man that I didn't have much time lately to continue working on it :'(

Things are crazy at my end now.

I'll close this PR

Reply:
Don't worry. I understand there are more important things That's all right.
Anyway your ideas and contributions are welcome :)